docker-compose.yml

```diff
@@ -48,6 +48,7 @@ services:
       - "44920:44920/udp"
       - "44930:44930/udp"
       - "44940:44940/udp"
+      - "44990:44990/udp"
```

Comment:
Does this port need to be exposed on any of the other docker-compose files?

Reply:
It looks like this port should be opened in the confluent version of the docker-compose as well. Fixed in the latest update.